src/app/signup/page.tsx

```diff
@@ -35,42 +68,58 @@ const SignUp = () => {
             <h2 className="text-3xl mb-5 font-semibold">Create your account</h2>
           </div>
           <div className="flex items-center px-12 mb-10 w-full">
-            <form action="" className="w-full">
-              <div className="flex mb-10">
+            <form onSubmit={handleSubmit(onSubmit)} action="" className="w-full">
```

Comment:
I think we can go ahead and remove the `action=""` attribute here now.

Reply:
Gone